tools/functional-tester/etcd-tester/tester.go

```diff
@@ -235,3 +257,25 @@ func isSameValueInMap(hashes map[string]int64) bool {
 	}
 	return ok
 }
+
+func (c *cluster) compactKV(rev int64) error {
+	var errMsg error
```

Comment:
We want to return error if all endpoints failed. Any thoughts? Mine is wrong, but it will be like:

```go
func (c *cluster) compactKV(rev int64) error {
	var errMsg error
	for _, u := range c.GRPCURLs {
		conn, err := grpc.Dial(u, grpc.WithInsecure(), grpc.WithTimeout(5*time.Second))
		if err != nil {
			errMsg = err
			continue
		}
		kvc := pb.NewKVClient(conn)
		ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
		_, err = kvc.Compact(ctx, &pb.CompactionRequest{Revision: rev})
		if err != nil {
			errMsg = err
			continue
		}
		cancel()
		return nil
	}
	return errMsg
}
```